lighthouse-cli/test/smokehouse/test-definitions/oopif-requests.js

```diff
@@ -47,11 +47,20 @@ const expectations = {
       'network-requests': {
         details: {
           items: {
-          // We want to make sure we are finding the iframe's requests (paulirish.com) *AND*
-          // the iframe's iframe's iframe's requests (youtube.com/doubleclick/etc).
-          // - paulirish.com ~40-60 requests
-          // - paulirish.com + all descendant iframes ~80-90 requests
-            length: '>70',
+            // We want to make sure we are finding the iframe's requests (paulirish.com) *AND*
+            // the iframe's iframe's iframe's requests (youtube.com/doubleclick/etc).
+            _includes: [
+              {url: 'http://localhost:10200/oopif-requests.html', finished: true, statusCode: 200, resourceType: 'Document'},
+              {url: 'https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/', finished: true, statusCode: 200, resourceType: 'Document'},
+              // FYI: Youtube has a ServiceWorker which sometimes cancels the document request. As a result, there will sometimes be multiple requests for this file.
+              {url: 'https://www.youtube.com/embed/NZelrwd_iRs', finished: true, statusCode: 200, resourceType: 'Document'},
+              // Subresource of youtube embed
+              {url: /https:\/\/www\.youtube\.com\/.*?\/embed.js/, finished: true, statusCode: 200, resourceType: 'Script'},
```

Comment:
why have all the finished/statusCode/resourceType stuff?

Reply:
generally: why not?

specifically:

**finished** tells us it ... finished. When working on this I was seeing cases where in an iframe we didn't get the `loadingFinished` event for that document.

However a request that gets `Network.loadingFailed` is marked as finished and thus the check of **statusCode**. it's -1 in bad cases.

and **resourceType** is there to basically verify it was transferred the way we expect. i don't expect it to catch much, but i don't see a reason why not.